infrastructure/ssz/src/test/java/tech/pegasys/teku/infrastructure/ssz/collections/SszBitvectorTest.java

```diff
@@ -239,4 +240,13 @@ void testGetLastSetBitIndex(final SszBitvector bitvector) {
   void get_shouldThrowIndexOutOfBounds(final SszBitvector vector) {
     assertThatThrownBy(() -> vector.get(-1)).isInstanceOf(IndexOutOfBoundsException.class);
   }
+
+  @ParameterizedTest
+  @MethodSource("bitvectorArgs")
+  void testFromBytes(final SszBitvector bitvector) {
+    Bytes bytes = bitvector.sszSerialize();
+    long size = bitvector.getSchema().getMaxLength();
+    SszBitvector result = SszBitvectorImpl.fromBytes(bitvector.getSchema(), bytes, (int) size);
```

Comment:
smallest of nits, lets make these variables final and then we're good here i think...

Reply:
can always fix the whole file, they should all really be final :)